### PR TITLE
 improve about  i18n: use short key and fallback mechanism

### PR DESCRIPTION
**Pull request by wuxiangwu**
improve about  i18n: use short key and fallback mechanism

### CODE REVIEW

File: content/locale/index.js [hunk 1]
```diff
@@ -47,3 +47,37 @@ function getLocale(lang) {
 			return 'en'
 	}
 }
+
+const isPlainObject = obj => {
+	return Object.prototype.toString.call(obj) === '[object Object]'
+}
+
+export function fallback() {
```
Comment (ahnafalfariza) [inline content/locale/index.js:55]: en.js
```
{
    a-key: 'Message for A'
    b-key: 'Message for B'
}
```
fr.js
```
{
    a-key: 'Translation for A'
}
```
this is handling fallback for a case like this right?

Comment (wuxiangwu) [reply content/locale/index.js:55]: yes right!

Comment (wuxiangwu) [reply content/locale/index.js:55]: just the en.js's b-key will add to fr.js.

Comment (wuxiangwu) [reply content/locale/index.js:55]: > btw is there any logs or information when the language (fr.js) does not provide the translation?
Hi ahnafalfariza,
just the b-key and the value of en.js will add to fr.js.

**Comment by wuxiangwu** [comment]
> also, I have additional request, I think it's better to make the keys in camelcase, ex: `PutOnSaleFor` the others are good btw. thanks, looking forward to it

Okay ,I will remvoe the '_' laterly

**Comment by wuxiangwu** [comment]
> > also, I have additional request, I think it's better to make the keys in camelcase, ex: `PutOnSaleFor` the others are good btw. thanks, looking forward to it
> 
> Okay ,I will remvoe the '_' laterly

Done, check it please! tks

**Pull request merged**